src/components/Thanks/ThanksPageSingleVersion.vue

```diff
@@ -27,8 +27,16 @@
 				@continue-clicked="handleContinue"
 				class="tw-mb-2.5"
 			/>
+			<GoalCompleted
```

Comment:
Add a story in the storybook file for this use case, this component is pretty well covered in storybook.

Reply:
We can resolve in follow-up PR.